src/command_line_parser.cc

```diff
@@ -1115,6 +1161,19 @@ TritonServerParameters::BuildTritonServerOptions()
   return managed_ptr;
 }
 
+std::string
+TritonParser::GetOptionName(const struct option* long_option)
+{
+  if (long_option != nullptr) {
+#ifdef _WIN32
+    return long_option->name_;
```

Comment:
Since `option` is supposed to be the mirror implementation for Windows, I think the `_` suffix can be dropped variable name to be identical as the linux implementation and `_WIN32` conditional can be droppd